### PR TITLE
Laravel 8 support + PHP7.4 version testing addition in Travis CI

### PR DESCRIPTION
**Pull request by Okipa**
* Added Laravel 8 support.
* Added PHP7.4 version testing in Travis CI.

### CODE REVIEW

**Comment by Okipa** [comment]
Hi @atymic (seems that you are actively maintaining this projet), I don't know why Scrutinizer is broken and why Travis is not triggered but here is my proposal for Laravel 8 support.

Could you look into it ?

File: composer.json [hunk 1]
```diff
@@ -11,16 +11,16 @@
         }
     ],
     "require": {
-        "php": "^7.2.5",
-        "guzzlehttp/guzzle": "~6.0",
-        "illuminate/notifications": "~5.5 || ~6.0 || ~7.0 || ^8.0",
-        "illuminate/support": "~5.5 || ~6.0 || ~7.0 || ^8.0"
+        "php": "^7.3",
+        "guzzlehttp/guzzle": "~7.0",
+        "illuminate/notifications": "~6.0 || ~7.0 || ^8.0",
```
Comment (chrysanthos) [inline composer.json]: If we are dropping support for L6, maybe the `~6.0` need to be removed as well?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ All notable changes to `webhook` will be documented in this file
 ## 2.1.0 - 2020-08-21
 
 * Added Laravel 8 support.
+* Dropped Laravel 6 support.
+* Dropped PHP7.2 support.
```
Comment (chrysanthos) [inline CHANGELOG.md]: Laravel 7 supports PHP >= 7.2.5. I dont think we can drop support for it now.

File: composer.json [hunk 3]
```diff
@@ -11,16 +11,16 @@
...
+        "illuminate/support": "~6.0 || ~7.0 || ^8.0"
     },
     "require-dev": {
         "mockery/mockery": "^1.3",
-        "phpunit/phpunit": "^8.0",
-        "orchestra/testbench": "^3.8.0 || ^4.0 || ^5.0 || ^6.0",
-        "orchestra/database": "^3.8.0 || ^4.0 | ^5.0 || ^6.0"
+        "phpunit/phpunit": "^9.0",
+        "orchestra/testbench": "^4.0 || ^5.0 || ^6.0",
```
Comment (chrysanthos) [inline composer.json]:
```suggestion
        "orchestra/testbench": "^5.0 || ^6.0",
```

File: composer.json [hunk 4]
```diff
@@ -11,16 +11,16 @@
...
+        "orchestra/database": "^4.0 | ^5.0 || ^6.0"
```
Comment (chrysanthos) [inline composer.json]:
```suggestion
        "orchestra/database": "^5.0 || ^6.0"
```

File: composer.json (hunk 3 above)
Comment (chrysanthos) [reply composer.json]: version 4.0 is for Larave 6

**Review by chrysanthos — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (Okipa) [reply composer.json]: Oops, you're right. Fixed !

File: composer.json (hunk 3 above)
Comment (Okipa) [reply composer.json]: Oops again. Also fixed !

File: CHANGELOG.md (hunk 2 above)
Comment (Okipa) [reply CHANGELOG.md]: Fixed !

**Comment by Okipa** [comment]
Thanks for reviewing @chrysanthos.

It's all fixed now.

**Comment by atymic** [comment]
Tagged
https://github.com/laravel-notification-channels/webhook/releases/tag/2.1.0

**Pull request merged**